File: src/com/peyo/lbsettings/MenuItem.java
package com.peyo.lbsettings;

import android.content.Context;

import com.google.gson.annotations.SerializedName;

public class MenuItem {
    @SerializedName("icon") private String mIcon = "";
    @SerializedName("title") private String mTitle = "";

    public int getIconRes(Context context) {
        return context.getResources().getIdentifier(mIcon, "drawable",
                context.getPackageName());
    }
    public int getTitleRes(Context context) {
        return context.getResources().getIdentifier(mTitle, "string",
                context.getPackageName());
    }

}

File: src/com/peyo/lbsettings/MenuItem.kt
package com.peyo.lbsettings

import android.content.Context

import com.google.gson.annotations.SerializedName

class MenuItem {
    @SerializedName("icon")
    private val mIcon = ""
    @SerializedName("title")
    private val mTitle = ""

    fun getIconRes(context: Context): Int {
        return context.resources.getIdentifier(mIcon, "drawable", context.packageName)
    }

    fun getTitleRes(context: Context): Int {
        return context.resources.getIdentifier(mTitle, "string", context.packageName)
    }

}
